galaxy/templates/rbac-fix-cvmfs.yaml

```diff
@@ -0,0 +1,42 @@
+{{- if and .Values.cvmfs.deployPostInstallFix (not .Values.cvmfs.deploy) .Values.refdata.enabled (eq .Values.refdata.type "cvmfs") }}
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: {{ include "galaxy.fullname" . }}-cvmfs-fix
+  namespace: {{ .Release.Namespace }}
+  labels:
+    {{- include "galaxy.labels" . | nindent 4 }}
+  {{- with .Values.serviceAccount.annotations }}
+  annotations:
+    {{- toYaml . | nindent 4 }}
+  {{- end }}
+---
+kind: Role
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  namespace: {{ .Values.cvmfs.externalCvmfsNamespace }}
+  name: {{ include "galaxy.fullname" . }}-cvmfs-fix
+  labels:
+    {{- include "galaxy.labels" . | nindent 4 }}
+rules:
+- apiGroups: [""]
+  resources: ["pods"]
+  verbs: ["get", "list","watch","delete"]
+---
+kind: RoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: {{ include "galaxy.fullname" . }}-cvmfs-fix
+  namespace: {{ .Values.cvmfs.externalCvmfsNamespace }}
```

Comment:
In general, I think it's preferable to constrain the helm chart to a single namespace. We don't want to expand the security footprint where it reaches across to other namespaces. Therefore, I think that if someone wants to create this service account, they do so manually, and we just document it.